AEPAssurance.podspec

```diff
@@ -0,0 +1,27 @@
+Pod::Spec.new do |s|
+  s.name         = "AEPAssurance"
+  s.version      = "1.0.0"
+  s.summary      = "AEPAssurance SDK for Adobe Experience Cloud. Written and maintained by Adobe."
+  s.description  = <<-DESC
+                   AEPAssurance SDK provides APIs that allow use of the AEPAssurance product in conjunction with the Adobe Experience platform solutions.
+                   DESC
+
+  s.homepage     = "https://git.corp.adobe.com/dms-mobile/v5-podspecs"
+
+  s.license      = {:type => "Commercial", :text => "Adobe.  All Rights Reserved."}
```

Comment:
Include the License file and also update the other fields as the other extensions.